index.bs

```diff
@@ -1341,7 +1341,8 @@ interface MLGraphBuilder {
   MLOperand input(USVString name, MLOperandDescriptor descriptor);
 
   // Create an operand for a graph constant.
-  MLOperand constant(MLOperandDescriptor descriptor, ArrayBufferView bufferView);
+  MLOperand constant(MLOperandDescriptor descriptor,
+                     AllowSharedBufferSource bufferSource);
```

Comment:
What do you think about just using `buffer`, at least in the IDL?
```suggestion
                     AllowSharedBufferSource buffer);
```

Reply:
FWIW `MLNamedArrayBufferViews` are slated to be removed alongside `compute()` #791

Reply:
Thanks for the suggestion and discussion! The latest commit uses `buffer`.